components/src/preact/mutationsOverTime/mutations-over-time-grid.tsx

```diff
@@ -18,49 +17,51 @@ export interface MutationsOverTimeGridProps {
 const MAX_NUMBER_OF_GRID_ROWS = 100;
 
 const MutationsOverTimeGrid: FunctionComponent<MutationsOverTimeGridProps> = ({ data }) => {
-    const mutations = data.getFirstAxisKeys();
-    if (mutations.length > MAX_NUMBER_OF_GRID_ROWS) {
-        throw new UserFacingError(
-            'Too many mutations',
-            `The dataset contains ${mutations.length} mutations. ` +
-                `Please adapt the filters to reduce the number to below ${MAX_NUMBER_OF_GRID_ROWS}.`,
-        );
-    }
+    const allMutations = data.getFirstAxisKeys();
+    const shownMutations = allMutations.slice(0, MAX_NUMBER_OF_GRID_ROWS);
 
     const dates = data.getSecondAxisKeys().sort((a, b) => compareTemporal(a, b));
 
     return (
-        <div
-            style={{
-                display: 'grid',
-                gridTemplateRows: `repeat(${mutations.length}, 24px)`,
-                gridTemplateColumns: `8rem repeat(${dates.length}, minmax(1.5rem, 1fr))`,
-            }}
-        >
-            {mutations.map((mutation, i) => {
-                return (
-                    <Fragment key={`fragment-${mutation.toString()}`}>
-                        <div
-                            key={`mutation-${mutation.toString()}`}
-                            style={{ gridRowStart: i + 1, gridColumnStart: 1 }}
-                        >
-                            <MutationCell mutation={mutation} />
-                        </div>
-                        {dates.map((date, j) => {
-                            const value = data.get(mutation, date) ?? { proportion: 0, count: 0 };
-                            return (
-                                <div
-                                    style={{ gridRowStart: i + 1, gridColumnStart: j + 2 }}
-                                    key={`${mutation.toString()}-${date.toString()}`}
-                                >
-                                    <ProportionCell value={value} date={date} mutation={mutation} />
-                                </div>
-                            );
-                        })}
-                    </Fragment>
-                );
-            })}
-        </div>
+        <>
+            {allMutations.length > MAX_NUMBER_OF_GRID_ROWS && (
+                <div>
```

Comment:
I would favour a little padding to the left, so the text seems more aligned with the table.
```suggestion
                <div className='pl-2'>
```